atomic_reactor/inner.py

```diff
@@ -282,8 +284,18 @@ def add_floating_images(self, images):
         for image in images:
             self.add_floating_image(image)
 
+    def load(self, data: Dict[str, Any]) -> None:
+        pass
```

Comment:
Maybe load should be a classmethod?
```suggestion
    @classmethod
    def load(cls, data: Dict[str, Any]) -> 'TagConf':
        pass
```
Not sure what your plan for loading is yet, but it seems like it could be difficult to use `load()` as an instance method (how would you create the instance before knowing the data to initialize it with?)